plugins/binja_binsync/binja_binsync.py

```diff
@@ -228,7 +235,7 @@ def push_stack_variable(self, bn_func: binaryninja.Function, stack_var: binaryni
                           type_str,
                           size,
                           bn_func.start)
-        state.set_stack_variable(bn_func.start, stack_var.storage, v)
+        state.set_stack_variable(v, bn_func.start, stack_var.storage)
```

Comment:
set_stack_variable args are: `variable, offset, func_addr`. Just confirming you have the right order here? Since it looks like you have the offset last here.